src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,11 +103,9 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setPreferredWidth(20);
-			getTable().getColumnExt(1).setPreferredWidth(50);
-			getTable().getColumnExt(2).setMaxWidth(30);
-			getTable().getColumnExt(3).setPreferredWidth(20);
+			getTable().getColumnExt(0).setMaxWidth(70);
```

Comment:
I'll try [`sizeWidthToFit()`](http://docs.oracle.com/javase/6/docs/api/javax/swing/table/TableColumn.html?is-external=true#sizeWidthToFit%28%29). Will post a reply or new code tonight.